### ISSUE TITLE
Add other Outputs

### ISSUE THREAD

**Issue by AbdelghaniKABOT** [ultralytics/ultralytics#1583]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar feature requests.

### Description

I hope you can add the option for the model to predict additional outputs, not just the class ID and bounding boxes. For example, if I want to detect the level of truncation and occlusion and these attributes are already present in my custom dataset, what should I do?

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by glenn-jocher** [comment]
@AbdelghaniKABOT you'd have to make changes to the dataloader, loss function, results outputs etc. to handle additional regression outputs.

**Comment by AbdelghaniKABOT** [comment]
@glenn-jocher Is there any documentation for this ?

**Comment by glenn-jocher** [comment]
@AbdelghaniKABOT no, unfortunately you're going to need to get your hands dirty customizing the code, we don't have Docs for this yet, but we are working hard to improve the Docs.
